static/templates/settings/organization_permissions_admin.hbs

```diff
@@ -161,17 +158,12 @@
                             <option value="{{0}}">{{1.text}}</option>
                         {{/each}}
                     </select>
-                </div>
-                <div class="dependent-block">
-                    <label for="id_realm_message_content_edit_limit_minutes" class="inline-block realm-time-limit-label">
-                        {{t 'Duration editing is allowed after posting (minutes)'}}:&nbsp;
-                    </label>
-                    <input type="text" id="id_realm_message_content_edit_limit_minutes"
-                      name="realm_message_content_edit_limit_minutes"
-                      class="admin-realm-time-limit-input prop-element"
-                      autocomplete="off"
-                      value="{{ realm_message_content_edit_limit_minutes }}"
-                      {{#unless realm_allow_message_editing}}disabled{{/unless}}/>
+                    <div class="dependent-block">
+                        <label for="id_realm_message_content_edit_limit_minutes" class="inline-block realm-time-limit-label">
+                            {{t 'Duration editing is allowed after posting (minutes)'}}:&nbsp;
+                        </label>
+                        <input type="text" id="id_realm_message_content_edit_limit_minutes" name="realm_message_content_edit_limit_minutes" class="admin-realm-time-limit-input prop-element" autocomplete="off" value="{{ realm_message_content_edit_limit_minutes }}" {{#unless realm_allow_message_editing}}disabled{{/unless}}/>
```

Comment:
Ca you undo the line-wrapping changes for the `input` elements? I think it's substantially less readable this way.

Reply:
Fixed